data/json/items/armor.json

```diff
@@ -165,7 +165,7 @@
         "material" : ["cotton", "leather"],
         "volume": 6,
         "warmth": 20,
-        "encumbrance": 10,
+        "encumbrance": 15,
```

Comment:
There is no need to nerf already useless items. Cleats don't have any real use at the moment.